### PR TITLE
Add check for animal participants

### LINKED ISSUE #672: Change metadata checker to check for animals on wwwadmin
If metadata pulldown fails for human, check for animal.  Only raise "could not pull down metadata" error if both fail.

### PR DESCRIPTION
**Pull request by adamrp**
Metadata checker on wwwadmin now checks for human and (if that fails) animal
metadata. Only reports failure if both of those attempts fail.

Fixes #672

### CODE REVIEW

**Comment by teravest** [comment]
the code looks good to me, do you have any barcodes we should test with.

**Comment by adamrp** [comment]
000009702 or 000005835 are animals that no longer raise the error and
display the correct light-green div indicating success.

**Comment by teravest** [comment]
This looks good to me.  @ElDeveloper will you please review tomorrow and a gif and then merge?

File: www/wwwadmin/barcode_util.psp [hunk 1]
```diff
@@ -142,8 +142,22 @@ try:
...
+                    div_id = "verified_animal"
+                    message = "All good"
+                    email_type = "1"
+                else:
+                    # should never get here (this would happen if the metadata
+                    # pulldown returned more than one row for a single barcode)
+                    div_id = "md_pulldown_error"
+                    message = ("This barcode has multiple entries in the "
+                               "database, which should never happeen. Please "
+                               "notify someone on the database crew.")
```
Comment (ElDeveloper) [inline www/wwwadmin/barcode_util.psp:160]: niiice

**Comment by ElDeveloper** [comment]
Such pull request:

t7auiyv

**Comment by ElDeveloper** [comment]
Thanks @adamrp and @teravest.

**Pull request merged**